### ISSUE TITLE
When I executing evalue.py, there will be a segmentation fault error

### ISSUE THREAD

**Issue by tian961214** [andr345/frtm-vos#14]
done
Evaluating dv2017val
bike-packing:   0%|                                                                                                                                       | 0/69 [00:00<?, ?frames/s]Segmentation fault

**Comment by andr345** [comment]
If python crashes, it can be something with your setup.
Can you find the file and line where you get the segmentation fault?

**Comment by tian961214** [comment]
hi!&nbsp; Thanks for your reply, I have solved this problem.&nbsp;
But I encountered a new problem in your STM paper, that is, how to pre-train on static images, what is the static image data set used,&nbsp;
and what is the training strategy. If this does not involve your privacy, please take the time to reply to me.


Thanks !!






------------------

 


田朗

北京邮电大学/研究生/信息与通信工程学院



13940216682

北京


 

&nbsp;
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------

**Comment by andr345** [comment]
STM is not ours though, so I can't help you much there.

But I think they are using COCO, Pascal, and these two datasets:
http://mmcheng.net/SalObj/
http://www.cs.cuhk.edu.hk/~leojia/projects/hsaliency/dataset.html

For training, I would make 3-frame synthetic sequences (like they did), with random affine transforms. You could try the data augmentation code in our repo for that.

Since they say nothing else (in section 3.5 "two-stage training" their paper), I would assume that they use the same optimizer, loss, schedule and other hyper parameters, in both pre-training and the regular one.
